flatdata-generator/tests/generators/go_expectations/archives/subarchive.go

```diff
@@ -59,8 +59,6 @@ func (v *XArchive) ToString() string {
     buffer := bytes.Buffer{}
     buffer.WriteString(fmt.Sprintf(`{"name": "X", "container_type": "Archive", "size_in_bytes": %d, "resources": [`, v.GetSizeInBytes()))
     buffer.WriteString(v.PayloadRawData.ToString())
-    buffer.WriteString(",")
```

Comment:
This seems like a behaviour change?

Reply:
Yes, this was the trailing comma bug in Go. See go template.